v2/lint/base.go

```diff
@@ -83,14 +83,19 @@ func (l *Lint) CheckEffective(c *x509.Certificate) bool {
 // CheckEffective()
 // Execute()
 func (l *Lint) Execute(cert *x509.Certificate) *LintResult {
-	if l.Source == CABFBaselineRequirements && !util.IsServerAuthCert(cert) {
+	if l.Source == CABFBaselineRequirements && !util.IsServerAuthCert(cert) && !util.IsDelegatedOCSPResponderCert(cert) {
+		// OCSP certificates also fall under certain BRG lints, even if they don't have the ServerAuth EKU set
```

Comment:
```suggestion
		// The Baseline Requirements only apply to those technically capable of TLS issuance, as well as the
		// infrastructure supporting them, such as OCSP responders. This is checked early, to avoid having to
		// duplicate this check in every lint.
```

This just tries to have the comment match the conditional, in terms of negation, as well as explain the early check.

This is mostly a minor comment.